src/components/InputLabel/InputLabel.tsx

```diff
@@ -0,0 +1,73 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
@SamuelAlev wouldn't it make more sense to have a `LICENSE` file or something like that? I'm not sure what benefit this line actually has and it's way to easy to forget :see_no_evil: